Moved text from :before to <button>

vinbaluyot opened the pull request:
<!-- Please read the contribution guide before contributing https://github.com/sButtons/sbuttons/blob/master/CONTRIBUTING.md -->
<!-- Please describe what changes or additions this pull request pertain to -->
Moved the button's text from `:before`'s `content` property to inside of the `<button>`.
_Side note: Please see my comment below regarding the compiler._
<!-- Specify the issue it relates to, if any --->
Issue: #803

shahednasser commented:
Hi @vinbaluyot there's no problem with the compiler, it just compiles everything (not based on what changed).

Can you please commit the changes after compiling? Changes in `dist/sbuttons.css` are necessary.

vinbaluyot commented:
> Hi @vinbaluyot there's no problem with the compiler, it just compiles everything (not based on what changed).
> 
> Can you please commit the changes after compiling? Changes in `dist/sbuttons.css` are necessary.

@shahednasser  It's done and I think I did it correctly this time. And of course `dist/sbuttons.css` is necessary. Silly me lol.

shahednasser reviewed (changes requested):
It looks great just a few remarks:

1. Include a rule for `.stack-btn.block-btn` where display is flex. I think you will need to add `!important` to work.
2. Change the margin for `.stack-btn.block-btn` to `2rem auto`

vinbaluyot commented:
> It looks great just a few remarks:
> 
> 1. Include a rule for `.stack-btn.block-btn` where display is flex. I think you will need to add `!important` to work.
> 2. Change the margin for `.stack-btn.block-btn` to `2rem auto`

To avoid using `!important`, I modified `.stack-btn`'s display from `flex` to `inline-flex` and added `display: flex` to `.stack-btn.block-btn` (also changed the margin as you mentioned above). Hope that works for you.

shahednasser commented on src/components/styles/_stack.less:
```diff
@@ -86,7 +103,8 @@
   height: 90%;
   width: 90%;
-  display: flex;
+  // display: flex;
```
Ok all looks great just remove this comment

vinbaluyot replied:
Ok the comment has been removed :)

shahednasser commented:
Great work thank you!

@all-contributors please add @vinbaluyot for code and bug
